src/main/java/com/base64/d/WriteToFile.java

```diff
@@ -3,51 +3,27 @@
 import java.io.FileWriter;
 import java.io.IOException;
 import java.io.PrintWriter;
-import java.io.*;
-import java.nio.file.Files;
-import java.nio.file.Paths;
-import java.nio.file.StandardOpenOption;
-import java.text.SimpleDateFormat;
 import java.util.Date;
 
-import static java.lang.System.out;
-
-
 public class WriteToFile {
 
-    String date = new Date().toString();
-
     public void createTxtFile(String titleOfStudyGuide, String link, String link2) {
+        String date = new Date().toString();
 
-
-        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
```

Comment:
Removing an unused variable. In IntelliJ the variable name will have a light grey color to indicate that it is not used. That's in the default color scheme, I'm not sure what the color is in the Darkula mode.